src/main/java/org/folio/rest/impl/LostItemFeePoliciesAPI.java

```diff
@@ -94,7 +105,33 @@ public void putLostItemFeesPoliciesByLostItemFeePolicyId(String lostItemFeePolic
     Handler<AsyncResult<Response>> asyncResultHandler,
     Context vertxContext) {
 
+    if (containsNegativeValue(entity)) {
+      handleError(entity, asyncResultHandler);
+      return;
+    }
+
     PgUtil.put(TABLE_NAME, entity, lostItemFeePolicyId, okapiHeaders, vertxContext,
       PutLostItemFeesPoliciesByLostItemFeePolicyIdResponse.class, asyncResultHandler);
   }
+
+  private void handleError(LostItemFeePolicy entity,
+    Handler<AsyncResult<Response>> asyncResultHandler) {
+
+      ImmutablePair<String, String> errorInformation = new ImmutablePair<>("lostItemProcessingFee",
+        getAmountValueAsString(entity));

```

Comment:
Why do we need this pair? Can these values be inlined?

Reply:
It is refactored, thanks!